Foundation/CGFloat.swift

```diff
@@ -279,33 +292,28 @@ extension CGFloat : BinaryFloatingPoint {
     }
 
     @_transparent
-    public var magnitude: CGFloat {
-        return CGFloat(Swift.abs(native))
-    }
-
-    @_transparent
     public mutating func negate() {
         native.negate()
     }
 
     @_transparent
-    public mutating func add(_ other: CGFloat) {
-        native.add(other.native)
+    public static func +=(_ lhs: inout CGFloat, _ rhs: CGFloat) {
```

Comment:
Technically not, but these methods were there only to implement operators on the protocol, which is no longer required. In any case these missing methods can now be added for source compatibility in the protocol. Thanks for looking at this!